client/js/lounge.js

```diff
@@ -293,6 +297,7 @@ $(function() {
 				networks: [data.network]
 			})
 		);
+		sidebar.find("#network-" + data.network.id).data("options", data.network.serverOptions);
```

Comment:
If any valid, [this comment](https://github.com/thelounge/lounge/pull/167/files#r58304321) applies here as well.